src/main/java/tatracker/model/ModelManager.java

```diff
@@ -214,6 +230,13 @@ public boolean hasGroup(Group group, Module targetModule) {
         return taTracker.hasGroup(group, targetModule);
     }
 
+
+    //=========== Filtered Student List Accessors =============================================================
+
+    /**
+     * Returns an unmodifiable view of the list of {@code Student} backed by the internal list of
+     * {@code versionedTaTracker}
+     */
```

Comment:
Oooh this must have been accidentally merged in during conflict resolution. It's not needed anymore!

Reply:
Okie, will delete in next PR